docs/contributing/style_guide.md

```diff
@@ -245,12 +245,13 @@ And not
 
 Code examples should not utilise the triple-back tick (\`\`\`) notation or tags.
 They should just be marked by indentation, which requires a break before the
-block and 6 additional spaces for each line of the example. This is 4 more than
-the first column of the comment. For example:
+block and 6 additional spaces for each line of the example. For example:
 
 ```ts
-/** A straight forward comment and an example:
```

Comment:
Its not really butifull if comments started at the first line
Could you please let me to change it ?

Reply:
@nayeemrmn ok, as you wish, but please use the same format for all documets.
I mean if you like comments to start at the first line, its ok but please let me to change all the comments to be same code style.
Am I right?